### DISCUSSION TITLE
求个指导，关于一个bug

### DISCUSSION THREAD

**Discussion by sqq135** [hooke007/mpv_PlayKit#512] [mpv 通用]
看剧的时候，切换下一集会黑屏，关不掉，只能进任务管理器强制终止。研究下发现是字幕问题，有两个字幕，srt字幕可以正常切换，ass字幕就会复现，大概是什么原因啊？已经看完了这部剧，就是想知道为什么那么奇怪，因为是强制停止log也不能导出，控制台在切换字幕时也没有报错。附上两个字幕文件。

[大臣原版DVD.zip](https://github.com/user-attachments/files/17352951/DVD.zip)

**Comment by hooke007** [comment]
无法复现

**Reply by sqq135** [reply]
我用夸克网盘分享了「大臣原版DVD (2).zip」，点击链接即可保存。打开「夸克APP」，无需下载在线播放视频，畅享原画5倍速，支持电视投屏。
链接：https://pan.quark.cn/s/59918b358f27
有空或者试一下？也不知哪个部分出问题了

这是mpv.conf中关于字幕的部分，去掉第一行sub-auto = fuzzy就正常了

 ########
 # 字幕 #
 ########

 sub-auto = fuzzy
 sub-file-paths = sub;subtitles
 slang = chs,sc,zh-cn,chi,zh,ch,cht,zho,ze,GB
 subs-fallback = yes
#blend-subtitles = yes

 sub-font = "华文中圆"
# sub-font-size = 44
#sub-scale-with-window = no
 sub-border-size = 4
 sub-blur = 1
 sub-color = "#FFD766"
#sub-back-color = "#641C1C1C"
#sub-shadow-offset = 2
 sub-use-margins = yes

#sub-ass-vsfilter-aspect-compat = no
 sub-ass-vsfilter-blur-compat = no
#sub-ass-override = force
#sub-ass-scale-with-window = yes
 sub-ass-force-margins = yes
#sub-ass-style-overrides = Fontname=LXGW WenKai Mono

stretch-image-subs-to-screen = yes
#image-subs-video-resolution = yes

**Reply by hooke007** [reply]
仍然正常

**Comment by sqq135** [comment]
有空我重新下载个新的lazy尝试复现，感觉是加载字幕的问题



---Original---
From: "Karl ***@***.***&gt;
Sent at: 2024年10月13日(Sun) Noon1:42
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hooke007/MPV_lazy] 求个指导，关于一个bug (Discussion #512)




 
仍然正常
